src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
```

Comment:
Based on the name you're using for the repository from which you're making the PR, I'm checking you for sure want to merge this to the definitions used for the IOXIO sandbox environment (sandbox.ioxio-dataspace.com) and not the Virtual Finland Testbed (testbed.fi).

In any case, a really big thanks for joining and welcome!

Reply:
Thanks for the detective work! I'll transfer this PR to a correct context.

Reply:
Also thank you for the comments! The intention of this PR was to just tinker around with something to get the hang of the data flow. The request input/output requirements and fields should be thought with a more thought with generalization/usability in mind, at the very least when moving towards a "draft".

Reply:
Great! Looking forward to the PR in the other repo :)

BTW I just checked, the guideline was indeed to namespace stuff under the `/test` directory, so please do move it in under a subfolder with your GitHub username when you create the PR in the right repo. And the comments I posted were just ideas/observations. When it comes to namespaced test definitions you're really free to experiment with whatever kind of definitions you want.